testing/src/network/topology.rs

```diff
@@ -93,7 +92,7 @@ async fn mesh(nodes: &mut Vec<Node<LedgerStorage>>) {
 }
 
 /// Connects the network nodes in a star topology.
-fn star(nodes: &mut Vec<Node<LedgerStorage>>) {
+async fn star(nodes: &mut Vec<Node>) {
```

Comment:
why the `async` here? there is no async code in this function

Reply:
consistency with siblings, but i don't really care so i'll change it